src/products/components/ProductVariantCreatorPage/ProductVariantCreatorSummary.tsx

```diff
@@ -303,6 +303,7 @@ const ProductVariantCreatorSummary: React.FC<ProductVariantCreatorSummaryProps>
               <div className={classNames(classes.col, classes.colSku)}>
                 <TextField
                   name="sku"
+                  data-test-id="sku-input"
```

Comment:
Can't we just get it by `cy.get('input[name="sku"]')`?

Reply:
Yes, I removed data test id